boards/x86/qemu_x86/board.cmake

```diff
@@ -56,6 +56,9 @@ endif()
 if(CONFIG_X86_SSE4A)
   string(JOIN "," QEMU_CPU_FLAGS "${QEMU_CPU_FLAGS}" "sse4a")
 endif()
+if((NOT CONFIG_X86_64) AND CONFIG_CACHE)
```

Comment:
Should we check for `CACHE_MANAGEMENT` ? I think braces are not needed in `NOT ...`

Reply:
CONFIG_CACHE is for driver, yes, not the actual api.